SubmittingPatches-backports.rst

```diff
@@ -334,14 +334,19 @@ Once the backport PR is open, the first order of business is to set the
 Milestone tag to the stable release the backport PR is targeting. For example,
 if the PR is targeting "nautilus", set the Milestone tag to "nautilus".
 
-If you don't have sufficient GitHub permissions to set the Milestone, add
-a comment to the PR with the following content::
-
-    @ceph/backport-admins
-
-This will alert the `Stable Releases and Backports team`_ to your PR, and
-someone will ensure your PR gets the proper labels.
-
+If you don't have sufficient GitHub permissions to set the Milestone, don't
+worry. Members of the `Stable Releases and Backports team`_ periodically run
+a script (``ceph-backport.sh --milestones``) which scans all PRs targetting stable
```

Comment:
s/targetting/targeting/

Reply:
yeah, I saw that, but figured we could just live with it... sorry

Reply:
ack, no worries!